### ISSUE TITLE
Importing event trigger crashes the provider

### ISSUE THREAD

**Issue by iochagov** [mongodb/terraform-provider-mongodbatlas#606]
### Terraform CLI and Terraform MongoDB Atlas Provider Version
```hcl
Terraform v1.0.11
on darwin_amd64
+ provider registry.terraform.io/hashicorp/aws v3.56.0
+ provider registry.terraform.io/hashicorp/random v3.1.0
+ provider registry.terraform.io/mongodb/mongodbatlas v1.0.2
+ provider registry.terraform.io/ojford/wireguard v0.1.3
```

### Terraform Configuration File 
```hcl
resource "mongodbatlas_event_trigger" "atlas_trigger" {
  project_id      = var.project_id
  app_id          = var.realm_app_id
  name            = "staging_chargebacks_to_s3_daily"
  type            = "SCHEDULED"
  function_id     = "1"
  disabled        = false
  config_schedule = "0 8 * * *"
}

```

### Steps to Reproduce
1. terraform init
2. terraform import mongodbatlas_event_trigger.atlas_trigger <group_id>--<app_id>--<trigger_id>

### Expected Behavior
The resource should be imported.

### Actual Behavior
Provider crashes.

### Debug Output 
[debug.log](https://github.com/mongodb/terraform-provider-mongodbatlas/files/7538367/debug.log)

### Crash Output
[crash.log](https://github.com/mongodb/terraform-provider-mongodbatlas/files/7538334/crash.log)

### Additional Context
I think not.

### References
No.

**Comment by themantissa** [comment]
Thank you @iochagov for the completed issue. We will have someone on the team take a look.

**Comment by themantissa** [comment]
Internal ticket INTMDB-279

**Comment by themantissa** [comment]
@iochagov the team has confirmed the bug and is working on a fix.  Thanks!

**Comment by themantissa** [comment]
@iochagov we are about to release the fix in version 1.2.  The pre-release is completed if you'd like to test first.  Thank you.

**Comment by themantissa** [comment]
This was addressed in version 1.2 of the provider.  Thank you!
